CHANGELOG.md

```diff
@@ -1,3 +1,93 @@
+
+## Fixes
+
+### Multiple Platforms
+
+- Fixed `TextField(TextFieldValue)` when used with a visual transformation with a non-identity offset mapping (potentially even crashing) [#2117](https://github.com/JetBrains/compose-multiplatform-core/pull/2117)
+- Fixed a memory leak and performance degradation when `ComposeUiFlags.isRectTrackingEnabled` set to `true` (default) [#2112](https://github.com/JetBrains/compose-multiplatform-core/pull/2112)
+- Support Preview parameters for Previews in common source sets in IJ and AS. Note: IDEs also need to implement support on their end. Please check the respective IDE release notes to confirm this is supported [#5319](https://github.com/JetBrains/compose-multiplatform/pull/5319)
+  Example usage:
+  ```
+  import androidx.compose.runtime.Composable
+  import org.jetbrains.compose.ui.tooling.preview.Preview
+  import org.jetbrains.compose.ui.tooling.preview.PreviewParameter
+  import org.jetbrains.compose.ui.tooling.preview.PreviewParameterProvider
+  class MyPreviewParameterProvider : PreviewParameterProvider<String> {
+    override val values = sequenceOf("Hello, Compose!", "Hello, World!")
+  }
+  /**
+   * This function will generate two preview images with different texts
+   */
+  @Preview
+  @Composable
+  fun MyPreview(@PreviewParameter(MyPreviewParameterProvider::class) text: String) {
+    Text(text)
+  }
+  ```
+
+### iOS
+
+- Fix issue where keyboard would appear after second tap when text input session was intercepted [#2049](https://github.com/JetBrains/compose-multiplatform-core/pull/2049)
+- _(prerelease fix)_ Fix keyboard jumps when switching between text fields [#2062](https://github.com/JetBrains/compose-multiplatform-core/pull/2062)
+
+### Desktop
+
+- [macOS] Fix the background flashing when closing a window/dialog and an animation is running [#2058](https://github.com/JetBrains/compose-multiplatform-core/pull/2058)
+- [macOS; JBR] Fixed the current composition in a text field being duplicated into another text field when switching focus to it [#2026](https://github.com/JetBrains/compose-multiplatform-core/pull/2026)
+- [macOS] Fixed strange glyph being displayed in a text field if window becomes unfocused, then focused again while there's an active composition in the text field (after pressing e.g. backspace) [#2026](https://github.com/JetBrains/compose-multiplatform-core/pull/2026)
+- [macOS] Fix showing the input method toolbar before any text field becomes focused (on JBR only; other runtimes continue to be buggy) [#2047](https://github.com/JetBrains/compose-multiplatform-core/pull/2047)
+- Improved performance for `ComposePanel` with `System.setProperty("compose.swing.render.on.graphics", "true")` [#2097](https://github.com/JetBrains/compose-multiplatform-core/pull/2097)
+- [Linux] Fix `svgPainter` doesn't show any images [#2097](https://github.com/JetBrains/compose-multiplatform-core/pull/2097)
```

Comment:
It was a part of 1.8.1

Reply:
If so, how to be sure there are no other wrong lines?

Reply:
I will look

Reply:
It looks like a change from Skiko update. We updated it 2 times - in release and main branches, to different versions. So it can't be identified as cherry-pick.

I will add additional exclusion condition - if the text of the changes equals another change from the release branch - exclude it.

Reply:
Fixed, @terrakok, please review